sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala

```diff
@@ -245,15 +245,27 @@ class DataFrameReader private[sql](sparkSession: SparkSession) extends Logging {
         "read files of Hive data source directly.")
     }
 
+    if ((extraOptions.contains("path") || extraOptions.contains("paths")) && paths.nonEmpty) {
+      throw new AnalysisException("There is a 'path' or 'paths' option set and load() is called " +
+        "with path parameters. Either remove the option or put it into the load() parameters.")
```

Comment:
`Either remove the option or not put it into the load() parameters`?

Reply:
I followed the suggestion from https://github.com/apache/spark/pull/29328#discussion_r464203078, but let me know if the current message is still vague. Thanks!

Reply:
to make it clearer:
```
Either remove the path option if it's the same as the path parameter, or add it
to the load() parameter if you do want to read multiple paths.
```